_docs/community.md

```diff
@@ -54,76 +54,73 @@ which will produce something like the following...
 
 If you're looking for cost vs power using ApexCharts, then @plandregan has you covered
 
-```yaml
 type: custom:apexcharts-card
 experimental:
-color_threshold: true
+  color_threshold: true
 header:
-show: true
-floating: true
-title: Cost vs Power
+  show: true
+  floating: true
+  title: Cost vs Power
 graph_span: 24h
 show:
-last_updated: true
-loading: true
+  last_updated: true
+  loading: true
 apex_config:
-legend:
-show: false
-chart:
-zoom:
-type: x
-enabled: true
-autoScaleYaxis: false
-toolbar:
-show: true
-autoSelected: zoom
-xaxis.type: datetime
-fill:
-type: gradient
-gradient:
-shadeIntensity: 0.1
-opacityFrom: 0.3
-opacityTo: 1
-inverseColors: true
+  legend:
+    show: false
+  chart:
+    zoom:
+      type: x
+      enabled: true
+      autoScaleYaxis: false
+    toolbar:
+      show: true
+      autoSelected: zoom
+    xaxis.type: datetime
+  fill:
+    type: gradient
+    gradient:
+      shadeIntensity: 0.1
+      opacityFrom: 0.3
+      opacityTo: 1
+      inverseColors: true
 series:
-
-    entity: sensor.octopus_energy_electricity_***************************_current_rate
+  - entity: sensor.octopus_energy_electricity_xxxxxxxxxxxx_1100000946808_current_rate
     transform: return x * 100;
     type: area
     name: GBP/kWh
     yaxis_id: pence
     color: lightblue
     group_by:
-    func: avg
-    duration: 5m
+      func: avg
+      duration: 5m
     stroke_width: 2
     extend_to: now
     show:
-    extremas: false
-    header_color_threshold: true
-    entity: sensor.givtcp_grid_power
+      extremas: false
+      header_color_threshold: true
+  - entity: sensor.givtcp_ce2146g163_grid_power
```

Comment:
Are you ok for this identifier to appear?

Reply:
thanks. missed that. xxxxx out the ce2146g163 if you don't mind.